### ISSUE TITLE
poylmer build, css minify, issue with the @apply

### ISSUE THREAD

**Issue by Menkhaus-ge** [Polymer/polymer-cli#890]
### Description: 
When the @apply is used in the css without a semicolon at the end and the polymer build has css minify set to true, the output css is not correctly minified. This issue causes the minified css to nest. 
These problems below are fixed if there is a semicolon that the end of the @apply line. However, the minifier does not require a semicolon at the end of the other lines of css. 

It is a separate issue that the node sass complier is stripping the semicolon from our sass. 

### Versions & Environment
polymer-build v2.0.0
polymer-cli v1.5.5
node v7.10.0

### Expected Results
Non-minified css
```
 .modal {
  position: fixed
  @apply --px-modal
}

.invisible {
  visibility: hidden
}
```
minified css
```
 .modal {
  position: fixed;
  @apply --px-modal;
}

.invisible {
  visibility: hidden;
}
```

#### @apply with no semicolon and defined last
Causes weird nesting
Non-minified css
```
 .modal {
  position: fixed
  @apply --px-modal
}

.invisible {
  visibility: hidden
}
```
minified css
```
 .modal {
  position: fixed;
  @apply --px-modal;
  .invisible {
    visibility: hidden;
   }
}
```

####  @apply with no semicolon and defined in the middle
Does not treat the @apply as a separate line
Non-minified css
```
 .modal {
 @apply --px-modal
 position: fixed
}

.invisible {
  visibility: hidden
}
```
minified css
```
.modal {
 @apply --px-modal  position: fixed;
}

.invisible {
  visibility: hidden;
}
```

####  @apply with semicolon and defined last
Behaves as expected
Non-minified css
```
 .modal {
  position: fixed
  @apply --px-modal;
}

.invisible {
  visibility: hidden
}
```
minified css
```
 .modal {
  position: fixed;
  @apply --px-modal;
}

.invisible {
  visibility: hidden;
}
```

**Comment by web-padawan** [comment]
You can only ommit semicilon in the last declaration, therefore you must place it after `position: fixed`.

**Comment by Menkhaus-ge** [comment]
My examples must have been misleading. From what I'm seeing the only line that seems to matter that as a semicolon or not is the @apply line
unminified css
```
.modal__title {
        margin-bottom: 1.33333rem;
        margin-top: 0;
        @apply --px-modal-title
      }

      .modal__buttons{
        @apply --px-modal-buttons
      }

      .modal__buttons--margin{
        margin-right: .66667rem
      }

      .modal__buttons--positive{
        @apply --px-modal-positive-button;
      }
```
minified css 
```
.modal__title {
          margin-bottom: 1.33333rem;
          margin-top: 0;
          @apply --px-modal-title;
          .modal__buttons {
            @apply --px-modal-buttons;
            .modal__buttons--margin {
              margin-right: .66667rem;
            }
            .modal__buttons--positive {
              @apply --px-modal-positive-button;
            }
          }
        }
```

**Comment by TimvdLippe** [comment]
I encountered this issue as well. I think this is an issue for css-slam

**Comment by Menkhaus-ge** [comment]
This issue is related to https://github.com/PolymerLabs/css-slam/issues/1

**Comment by azakus** [comment]
Should be fixed with css-slam 2.0.1. Please reopen if this still doesn't work for you.

**Comment by Menkhaus-ge** [comment]
Thank you for your help with this. 
The polymer-cli is still loading `"css-slam": "^1.1.0",` in its package.json
Can you bump this version to pick up the fix?
